thirdparty/kyaml/runfn/runfn.go

```diff
@@ -152,38 +131,21 @@ func (r RunFns) getNodesAndFilters() (
 		return nil, nil, outputPkg, err
 	}
 
-	fltrs, err := r.getFilters(buff.Nodes)
+	fltrs, err := r.getFilters()
```

Comment:
One small detail I realized just now (from `fn render` design):

We filter out the function configs from the input resources and add them only if `--include-metaresources` is enabled.

We want to do something similar here to ensure consistent behavior.

Reply:
Added a step to filter out the function config file.